general/administration/reverse-proxy.md

```diff
@@ -8,7 +8,7 @@ title: Reverse Proxy
 It's possible to run Jellyfin behind another server acting as a reverse proxy.  With a reverse proxy setup, this server handles all network traffic and proxies it back to Jellyfin. This provides the benefits of using DNS names and not having to remember port numbers, as well as easier integration and management of SSL certificates.
 
 Some popular options for reverse proxy systems are [Apache](https://httpd.apache.org/), [Haproxy](https://www.haproxy.com/), [Nginx](https://www.nginx.com/), [Caddy](https://caddyserver.com/) and [Traefik](https://traefik.io/).
-
+ 
```

Comment:
why add a space?

Reply:
mistake, if you remove it here I'll commit with the rest

Reply:
```suggestion

```